packages/commerce-sdk-react/README.md

```diff
@@ -6,6 +6,51 @@ A library of React hooks for fetching data from Salesforce B2C Commerce.
 
 The full documentation for PWA Kit and Managed Runtime is hosted on the [Salesforce Developers](https://developer.salesforce.com/docs/commerce/pwa-kit-managed-runtime/overview) portal.
 
+
+## PWA-Kit Integration
+
+> To integration this library with your PWA-Kit application you can use the `CommerceApiProvider` directly given that you use the `withReactQuery` higher order component to wrap your `AppConfig` component. Below is a snippet of how this is accomplished.
+
+```
+// app/components/_app-config/index.jsx
+
+import {QueryClient, QueryClientProvider} from '@tanstack/react-query'
+import {withReactQuery} from 'pwa-kit-react-sdk/ssr/universal/components/with-react-query'
+
+const AppConfig = ({children}) => {
+    return (
+        <CommerceApiProvider {...commerceApiProviderProps}>
+            {children}
+        </CommerceApiProvider>
+    )
+} 
+
+export defualt withReactQuery(AppConfig)
```

Comment:
Since the next section below mentions about `queryClientConfig`, let's also show here how they can pass in a config.